addons/repair/functions/fnc_doRepair.sqf

```diff
@@ -31,9 +31,23 @@ _hitPointDamage = _hitPointDamage max ([_unit] call FUNC(getPostRepairDamage));
 // raise event to set the new hitpoint damage
 ["setVehicleHitPointDamage", _vehicle, [_vehicle, _hitPoint, _hitPointDamage]] call EFUNC(common,targetEvent);
 
+// Repair the rest in the group (no specific damage, main hitpoint is enough)
+_hitpointGroup = configFile >> "CfgVehicles" >> typeOf _vehicle >> QGVAR(hitpointGroup);
+_hitpointGroup = if (isArray _hitpointGroup) then {getArray _hitpointGroup} else {[]};
+if (count _hitpointGroup > 0) then {
```

Comment:
I mean that you will end up resetting the damage for any hitpoint in the hitpoint group, even if the repair action is not for any hitpoint in the hitpoint group.